core/benchmarks/ZStandardBenchmark-jdk21-results.txt

```diff
@@ -2,26 +2,26 @@
 Benchmark ZStandardCompressionCodec
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 21+35 on Linux 5.15.0-1046-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+OpenJDK 64-Bit Server VM 21.0.1+12-LTS on Linux 5.15.0-1050-azure
```

Comment:
Was the test result generated by 1.5.5-7?

Reply:
Yes, it's not the result of `1.5.5-6`.
JDK 17: https://github.com/panbingkun/spark/actions/runs/6700489234
JDK 21: https://github.com/panbingkun/spark/actions/runs/6700491773